### PR TITLE
update console output to specify post release steps

### LINKED ISSUE #102166: [flutter_conductor] Add checklist items to conductor output
Add prompts to complete post release items into the verify release section of the conductor output.

### PR DESCRIPTION
**Pull request by itsjustkevin**
Add post release steps to conductor output

cc: @CaseyHillers

### CODE REVIEW

File: dev/conductor/core/lib/src/state.dart [hunk 1]
```diff
@@ -180,7 +180,15 @@ String phaseInstructions(pb.ConductorState state) {
     case ReleasePhase.PUBLISH_CHANNEL:
       return 'Issue `conductor next` to publish your release to the release branch.';
     case ReleasePhase.VERIFY_RELEASE:
-      return 'Release archive packages must be verified on cloud storage: ${luciConsoleLink(state.releaseChannel, 'packaging')}';
+      return <String>[
```
Comment (christopherfujino) [inline dev/conductor/core/lib/src/state.dart:183]: I think you should either move this to the next phase `RELEASE_COMPLETED` (note we call this function at the end of the `conductor next` flow), or also add this text the prompt that we have the user acknowledge before proceeding.

Comment (christopherfujino) [reply dev/conductor/core/lib/src/state.dart:183]: Otherwise lgtm

Comment (itsjustkevin) [reply dev/conductor/core/lib/src/state.dart:198]: Moved instructions and updated prompt

**Review by christopherfujino — APPROVED** [review]
LGTM!

**Comment by itsjustkevin** [comment]
TODO: create logic to give specific instructions based on if the release is a beta or stable release.

**Review by christopherfujino — APPROVED** [review]
Still LGTM

**Pull request merged**